nuxeo-java-client/src/main/java/org/nuxeo/client/Operations.java

```diff
@@ -36,4 +36,18 @@
      */
     public static final String DOCUMENT_REMOVE_PROXIES = "Document.RemoveProxies";
 
+    /**
+     * This operation is available since Nuxeo 10.1.
+     * 
+     * @since 3.1
+     */
+    public static final String TRASH_DOCUMENT = "Document.Trash";
```

Comment:
Could you rename it to `DOCUMENT_TRASH` (same thing for untrash) and if make sens sort them in an alphabetical order.